defaults/main.yml

```diff
@@ -162,8 +160,7 @@ checkmk_server__webapi_url: '{{ checkmk_server__site_url + "/check_mk/webapi.py"
 # :ref:`checkmk_server__ref_omd_config` for more details.
 checkmk_server__omd_config: '{{
   checkmk_server__omd_config_email +
-  checkmk_server__omd_config_core +
-  (checkmk_server__omd_config_livestatus if checkmk_server__multisite_livestatus|d() else [])
+  checkmk_server__omd_config_core
 }}'
```

Comment:
It is much more flexible to make changes to a dict on various Ansible variable levels. That is what various core roles like debops.ferm debops.ifupdown are switching to or have already switched to. Refer to this example: https://github.com/debops-contrib/ansible-volkszaehler/blob/72f6a79c434d1d4c665a581506bc221507ee4608/defaults/main.yml#L383-L478